docs/guides/all-keys-one-secret.md

```diff
@@ -18,10 +18,60 @@ Now, when creating our ExternalSecret resource, instead of using the data field,
 ```yaml
 {% include 'gcpsm-data-from-external-secret.yaml' %}
 ```
+<br>
+Here, "example" is the name of Kubernetes secrets which will be created.
+<br>
+when we use 
 
-To check both values we can run:
+```
+  dataFrom:
+  - extract:
+      key: all-keys-example-secret
+```
+We get all the secrets present in the key-value pair in the secret manager(external secret store of GCP or AWS or Azure) in the form of key values and also we can pass either all or a few key-values as environment variables.
+<br>
```

Comment:
Please refrain from using html <br> tags here and stick to plai markdown where possible. Unless you need explicit spacings, which I don't think you need here

Reply:
I have removed all the <br> tags and used the standard markdown